llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -263,6 +263,10 @@ struct VPTransformState {
   /// \see set.
   Value *get(VPValue *Def, unsigned Part);
 
+  /// Get the generated Value for a given VPValue \p Def and a given \p Part or
+  /// lane 0 for the part if \p NeedsScalar is true. \see set.
```

Comment:
It's more a matter of whether the Value to get is vector or scalar, both being per Part, than the fact that the scalar is stored and retrieved in lane 0 of Part.

nit: should `\see` be `See`, here and above?

Reply:
Also added an assert to make sure there's exactly one scalar per part when `IsScalar` is true